README.md

```diff
@@ -54,25 +54,25 @@ To deploy workloads with SRIOV VF this plugin needs to work together with the fo
 
   - On Pod deletion, reset and release the VF from the Pod
 
-This implementation follows the design discuessed in [this proposal document](https://docs.google.com/document/d/1Ewe9Of84GkP0b2Q2PC0y9RVZNkN2WeVEagX9m99Nrzc/).
+This implementation follows the design discussed in [this proposal document](https://docs.google.com/document/d/1Ewe9Of84GkP0b2Q2PC0y9RVZNkN2WeVEagX9m99Nrzc/).
 
 
 Please follow the Multus [Quick Start](#quick-start) for multi network interface support in Kubernetes.
 
 ### Supported SRIOV NICs
 
 The following  NICs were tested with this implementation. However, other SRIOV capable NICs should work as well.
--  Intel® Ethernet Controller X710 Series 4x10G
+-  IntelÂ® Ethernet Controller X710 Series 4x10G
```

Comment:
I know these weren't part this change but would it be possible to fix the character encoding issues as well? I see there are changes on these lines too.
Thanks!